internal/cloud/realm/dependencies_test.go

```diff
@@ -28,8 +28,15 @@ func TestRealmDependencies(t *testing.T) {
 		assert.Nil(t, err)
 
 		uploadPath := filepath.Join(wd, "testdata/dependencies_upload.zip")
+
+		_, err = client.GetDependenciesStatus(groupID, app.ID)
+		assert.NotNilf(t, err, "dependency status must return error before import")
 		assert.Nil(t, client.ImportDependencies(groupID, app.ID, uploadPath))
 
+		status, err := client.GetDependenciesStatus(groupID, app.ID)
+		assert.Equalf(t, status, realm.DependenciesStatusSuccessful, "must wait until dependency status is successful during import")
```

Comment:
[optional] I don't think the extra message here adds much, feel free to just use `assert.Equal` here